src/en/tut-lxd.md

```diff
@@ -40,8 +45,8 @@ should already be the case but you can confirm this by running the command:
 groups
 ```
 
-Your groups may vary, but if `lxd` is absent you can add yourself to the `lxd`
-group with:
+Your groups may vary, but if `lxd` is absent you can update your group
+membership group with:
```

Comment:
this wasn't broken but now it is ;)

Reply:
Why is it broken? The `newgrp` command does not add a user to a group. It merely updates (refreshes) your shell so that it becomes aware of your actual group membership.

Reply:
Fine. I think the new explanation isn't well expressed is all. Perhaps just:

... if `lxd` is absent you can refresh your groups with:

Reply:
OK